bin/nodenv-each

```diff
@@ -48,11 +58,19 @@ for arg in "$@"; do
   shift
 done
 
+
 if [ $# -eq 0 ]; then
   nodenv-help --usage each >&2
   exit 1
 fi
 
+if [ -n "$parallel" ]; then
+    if ! command -v parallel &> /dev/null; then
+        echo "You must install the parallel tool in order to run jobs in parallel"
+        exit 1
```

Comment:
Thoughts on just printing the error message but allowing the command to proceed without parallelism?

Reply:
I'm not sure about that. If the user requests parallelism but we cannot do it, they should be informed on how to achieve the parallelism.